assets/dev/scss/frontend/frontend-lite.scss

```diff
@@ -25,9 +25,6 @@
 // Animate
 @import "animate";
 
-// Shapes
-@import "shapes";
```

Comment:
Could we move the shapes.scss file to a separate folder with conditionally loaded files?

Reply:
@hein-obox What do you think if we move all shapes-related files to a separate module?

Reply:
I approved the PR for now. I will let you decide if you want to do this now or in another task.

Reply:
@Batsirai-muchareva I will approve for now as well.
You can decide how to handle this.

@rami-elementor What do you have in mind? Which files would be part of that module? I am not sure how that module would be used though.

I was thinking something like: 
frontend/utils/shapes.scss
or
frontend/modules/shapes.scss

I am not sure what would be the best suitable folder name.

Reply:
Forget the module, let's focus on CSS optimization. I'm approving and merging this PR.